### ISSUE TITLE
Not clear how to show terminal hover with keyboard

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#27771]
Testing -188537

I can't figure out how to get a terminal hover to show up without using the mouse, so I don't understand how a screenreader user would use this. There is an editor-specific command for this, but maybe there should be one for the workbench? I also don't know how to move focus from the terminal into the tab sidebar
